### DISCUSSION TITLE
Beregner ikke medmindre jeg dobbelttrykker alt+b eller prøver gentagende gange

### DISCUSSION THREAD

**Discussion by Oliverxll** [Eduap-com/WordMat#70] [Support] [answered]
Hej
Har lige frisk installeret wordmat og jeg møder allerede en fejl.

Når jeg gerne vil bruge beregneren til f.eks. så simpelt et udtryk som 4^2+8^2 så viser den ikke resultatet medmindre jeg trykker alt+b mange gange, eller meget hurtigt dobbelt tapper b mens jeg holder alt inde.

Om det egentligt er en fejl eller ej det ved jeg ikke, jeg kan bare huske jeg altid har trykket alt+b en gang for beregn og ikke mange gange/dobbelt tryk

**Comment by Eduap-com** [comment]
Jeg har aldrig set dette problem før. Nogen der har oplevet noget lignende?

**Comment by soegaard** [comment]
@Oliverxll  Kan du evt. optage en lille video, der viser problemet?

**Comment by Oliverxll** [comment]
@Eduap-com @soegaard 

Hver gang musen viser loading hjulet, så er det fordi jeg har trykket alt+b. Virtuelt tastatur viser ikke jeg trykker på b, åbenbart, men pointen er der, jeg skal trykke mange gange

https://user-images.githubusercontent.com/31373796/183257403-2ea7d99f-c320-4570-ba24-117d906536e4.mp4

**Comment by Eduap-com** [comment]
Fungerer det hvis du vælger GeoGebra som CAS motor under indstillinger?
Har du en eller anden form for antivirus installeret?

**Comment by Oliverxll** [comment]
@Eduap-com Der virker det heller ikke. Jeg har kun Windows Defender installeret

**Comment by Eduap-com** [comment]
Det vil nok kræve en større undersøgelse af computeren for at afklare fejlen. Mit sidste bud er at geninstallere officepakken.
Hvis du finder løsningen må du meget gerne skrive den her, så andre kan få glæde af den:-)

**Comment by Oliverxll** [comment]
Geninstallering af office pakken har jeg allerede gjort, da jeg føler jeg har tømt mine muligheder for at diagnosticere det. Jeg har tænkt mig i løbet af de næste par dage at geninstallere windows helt så min computer er klar til studiestart, det bør fikse det :)
Jeg holder dig lige opdateret når det er

**Comment by Oliverxll** [comment, marked as answer]
En total geninstallering af Windows løste problemet. Så problemet kan vel antages at have været en systemfejl af en art
